satpy/utils.py

```diff
@@ -303,8 +303,11 @@ def get_satpos(dataset):
         try:
             alt = orb_params['satellite_actual_altitude']
         except KeyError:
-            alt = orb_params['projection_altitude']
-            warnings.warn('Satellite altitude not available, using projection altitude instead.')
+            try:
+                alt = orb_params['satellite_nominal_altitude']
```

Comment:
should we have a warning for this too ?

Reply:
I think we shouldn't warn when the nominal values are used because they usually don't deviate much from the actual value. But the projection values can differ significantly.